src/context/likes-context/likes-context.js

```diff
@@ -0,0 +1,95 @@
+import { useContext, useState, createContext } from "react";
+import { useEffect } from "react";
+import { useToast } from "../toast-context/toast-context";
+import axios from "axios";
+
+const LikesContext = createContext();
+
+const useLikes = () => useContext(LikesContext);
+
+function LikesProvider({ children }) {
+  const encodedToken = localStorage.getItem("smashTubeToken");
```

Comment:
suggestion: you should not be fetching it from localStorage from every page. Better to keep it at one place somewhere and use it.